src/Compilers/Core/Portable/Compilation/Compilation.cs

```diff
@@ -2832,5 +2832,14 @@ public static string GetRequiredLanguageVersion(Diagnostic diagnostic)
 
             return foundVersion;
         }
+
+        /// <summary>
+        /// Determine what kind of conversion, if any, there is between the types 
+        /// <paramref name="source"/> and <paramref name="destination"/>.
+        /// </summary>
+        public ConversionInfo ClassifyConversionInfo(ITypeSymbol source, ITypeSymbol destination)
```

Comment:
I considered calling this ClassifyConversion.  But that would cause a Source-Breaking-Change with the existing ClassifyConversion helpers.  that's because the existing helpers are *extension methods* at the C# and Vb layer on the Compilation type.  So if i added 'ClassifyConversion' that would match before the extension method would, and you'd now get the new type back.

As the new type is not source compatible with the old types, this would definitely break things.  

I'm not a huge fan of this name.  One name i'm considering is GetConversionInfo to match GetSymbolInfo/GetTypeInfo.  Thoughts?